src/tests/ftest/daos_test/suite.yaml

```diff
@@ -22,7 +22,7 @@ timeouts:
   test_daos_capability: 104
   test_daos_epoch_recovery: 104
   test_daos_md_replication: 104
-  test_daos_rebuild_simple: 1800
+  test_daos_rebuild_simple: 3600
```

Comment:
This is an extra 30 minutes for a PR test. Can this instead be broken into multiple test cases where one runs in PR and another in daily or weekly?